pkg/providers/amifamily/ami.go

```diff
@@ -119,7 +119,9 @@ func (p *DefaultProvider) List(ctx context.Context, nodeClass *v1beta1.EC2NodeCl
 	}
 	amis.Sort()
 	if p.cm.HasChanged(fmt.Sprintf("amis/%s", nodeClass.Name), amis) {
-		logging.FromContext(ctx).With("ids", amis, "count", len(amis)).Debugf("discovered amis")
+		logging.FromContext(ctx).With(
+			"ids", lo.Uniq(lo.Map(nodeClass.Status.AMIs, func(a v1beta1.AMI, _ int) string { return a.ID })),
+			"count", len(amis)).Debugf("discovered amis")
```

Comment:
This count value isn't going to match the id length

Reply:
Updated. Thinking about removing the count value, not sure of the value if we are not truncating anymore

Reply:
Yeah, there isn't really a value. I'd agree, we should just drop it.